### PR TITLE
generate: add OOM Killer disable support

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by liangchenye** [comment]
LGTM

File: cmd/ocitools/generate.go [hunk 1]
```diff
@@ -333,6 +334,10 @@ func setupSpec(g *generate.Generator, context *cli.Context) error {
 		g.AddLinuxGIDMapping(hid, cid, size)
 	}
 
+	if context.IsSet("oom-kill-disable") {
+		g.SetLinuxResourcesDisableOOMKiller(context.Bool("oom-kill-disable"))
```
Comment (wking) [inline cmd/ocitools/generate.go]: Follow the spec ordering and use `--disable-oom-killer`?

File: man/ocitools-generate.1.md [hunk 2]
```diff
@@ -159,6 +159,9 @@ read the configuration from `config.json`.
   using tools like setuid apps.  It is a good idea to run unprivileged
   containers with this flag.
 
+**--oom-kill-disable**=true|false
+  Whether to disable OOM Killer for the container or not..
```
Comment (wking) [inline man/ocitools-generate.1.md]: `..` → `.`.

File: cmd/ocitools/generate.go (hunk 1 above)
Comment (Mashimiao) [inline cmd/ocitools/generate.go]: On 09/13/2016 12:20 PM, W. Trevor King wrote:
> Follow the spec ordering and use |--disable-oom-killer|?
Fine, fixed.

**Comment by wking** [comment]
dd638c280 looks good to me.

**Comment by Mashimiao** [comment]
PR rebased.
Ping @mrunalp  @liangchenye

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**